backend/src/managers/UserManager.ts

```diff
@@ -696,4 +697,14 @@ export default class UserManager {
     async getPublicKey(userId: number) {
         return this.userCache.getPublicKey(userId);
     }
+
+    async getUserVisitedDaysAgo(userId: number): Promise<number> {
+        const lastVisited = await this.userRepository.getUserLastVisited(userId);
+
+        const daysAgo =  lastVisited ?
+            moment().diff(lastVisited, "days") : 0;
```

Comment:
returning `0` for users who never visited the site could be confusing. Maybe worth the effort to return `undefined`/`null` in this case and handle it in UI.

Reply:
Fixed, now we do not show any tooltip, if there are no user records in activity db.